messages/md.deployreport.json

```diff
@@ -11,7 +11,10 @@
     "verbose": "verbose output of deploy results",
     "jobId": "job ID of the deployment to check; required if you’ve never deployed using Salesforce CLI; if not specified, defaults to your most recent CLI deployment",
     "wait": "wait time for command to finish in minutes.  Use -1 to poll indefinitely",
-    "concise": "omit success messages for smaller JSON output"
+    "concise": "omit success messages for smaller JSON output",
+    "junit": "output JUnit test results",
+    "coverageFormatters": "output coverage results in the specified formatters",
+    "outputDir": "output directory for results"
```

Comment:
```suggestion
    "outputDir": "output directory for code coverage and JUnit results"
```

Reply:
This flag doesn't default to the deploy ID? If it DOES default to that, add "; defaults to the deploy ID" to the end.